src/components/Modal.tsx

```diff
@@ -54,7 +54,8 @@ const Modal: React.FC<Props> = ({
       document.getElementById('root')! || document.body.firstElementChild
     }
     ariaHideApp
-    role="none"
+    aria-modal
+    role="alertdialog"
```

Comment:
These seemed to be appropriate attribute values for a modal, though nothing in this PR actually requires them.

Reply:
If not required or not useful, wouldn't it be better to delete 'em?

Reply:
I could go either way. It’s not clear to me what the future of our ARIA markup should be. I’m okay removing it if you think we should.

Reply:
Ah right… we're not really using ARIA for anyone else other than ourselves. 

Well, I leave it to you to decide as perhaps you have the most recent experience understanding exactly what each of these does and how we benefit from them.